docs/.vuepress/components/BuildPage.vue

```diff
@@ -42,6 +42,7 @@
               <a
                 target="_blank"
                 rel="noopener noreferrer"
+                class="hide-icon"
```

Comment:
(Outside of the scope of this PR) - should these be secondary buttons instead of standard links?
![Image 2020-03-10 at 12 10 32 PM](https://user-images.githubusercontent.com/8097623/76350008-3c210680-62c8-11ea-9a1e-aa4c199ab3af.png)

Reply:
I don't think we should change these — cards would address them